dask_sql/physical/rex/core/call.py

```diff
@@ -235,7 +235,7 @@ def cast(self, operand, rex=None) -> SeriesOrScalar:
         # TODO: ideally we don't want to directly access the datetimes,
         # but Pandas can't truncate timezone datetimes and cuDF can't
         # truncate datetimes
-        if output_type == "DATE":
+        if output_type == "DATE" or output_type == "TIMESTAMP":
```

Comment:
Is there a specific test that prompted the change? The logic exists only for the `date` type because it truncates all time info and only keeps the date.

Reply:
accidentally left there from some experimentation. Removing it.